apps/myaccount/src/pages/overview.tsx

```diff
@@ -35,28 +37,67 @@ const OverviewPage = (): ReactElement => {
     const isProfileInfoLoading: boolean = useSelector( (state: AppState) => state.loaders.isProfileInfoLoading);
     const profileDetails: AuthStateInterface = useSelector((state: AppState) => state.authenticationInformation);
     const [ userProfileName, setUserProfileName ] = useState<string>(null);
+    const [ isFederatedUser, setIsFederatedUser ] = useState<boolean>(undefined);
+    const [ userSource, setUserSource ] = useState<string>(null);
 
     useEffect(() => {
-
         if (isProfileInfoLoading === undefined) {
             return;
         }
 
         setUserProfileName(resolveUserProfileName(profileDetails, isProfileInfoLoading));
     }, [ isProfileInfoLoading ]);
 
+    /**
+     * Checks if the user is a user without local credentials.
+     */
+    useEffect(() => {
+        if (profileDetails?.profileInfo?.[ProfileConstants.SCIM2_ENT_USER_SCHEMA]?.
+            [ProfileConstants?.SCIM2_SCHEMA_DICTIONARY.get("USER_ACCOUNT_TYPE")]) {
+            setIsFederatedUser(true);
+        }
+
+        if (profileDetails?.profileInfo?.[ProfileConstants.SCIM2_ENT_USER_SCHEMA]?.
+            [ProfileConstants?.SCIM2_SCHEMA_DICTIONARY.get("USER_SOURCE")]) {
+            setUserSource(profileDetails?.profileInfo?.[ProfileConstants.SCIM2_ENT_USER_SCHEMA]?.
+                [ProfileConstants?.SCIM2_SCHEMA_DICTIONARY.get("USER_SOURCE")]);
+        }
+
+    }, [profileDetails?.profileInfo]);
+
     return (
         <InnerPageLayout
             pageTitle={ userProfileName ? (
-                t(
-                    "myAccount:pages:overview.title",
-                    { firstName: userProfileName }
-                )
+                <div>
+                    {
+                        t(
+                            "myAccount:pages:overview.title",
+                            { firstName: userProfileName }
+                        )
+                    }
+                    { userSource && isFederatedUser &&
+                        <div className="overview-page-header">
+                            {
+                                "(Signed up via " + userSource + ")"
```

Comment:
Let's add i18n strings.
```suggestion
                                "(Signed up via " + userSource + ")"
```

Reply:
Sure. I will not commit this as this has been relocated to a different file. There I will use i18n string.